### PR TITLE
Feature/update prettier config

### PR DESCRIPTION
**Pull request by christianmemije**
## Description

Changes to prettier config options.

### What's included?

- `htmlWhitespaceSensitivity ` from `css` -> `ignore ` - `css` can lead to some weird formatting such as: 
```html
<span
  >Sample text.</span
>
```
In angular components, [`preservewhitespaces`](https://angular.io/api/core/Component#preservewhitespaces) defaults to `false`, so just default to false as well.
- Reference `yarn` instead of `npm` in the doc

#### Test Steps

- [ ] `npm run serve`
- [ ] then this
- [ ] finally this

#### General Tests for Every PR

- [ ] `npm run serve:prod` still works.
- [ ] `npm run tslint` passes.
- [ ] `npm run stylelint` passes.
- [ ] `npm test` passes and code coverage is not lower.
- [ ] `npm run build:release` still works.

### CODE REVIEW

File: prettier/prettier.config.js [hunk 1]
```diff
@@ -8,5 +8,5 @@ module.exports = {
   bracketSpacing: true,
   arrowParens: 'always',
   proseWrap: 'preserve',
-  htmlWhitespaceSensitivity: 'css',
-};
\ No newline at end of file
+  htmlWhitespaceSensitivity: 'ignore',
```
Comment (richavyas) [inline prettier/prettier.config.js:11]: Do we plan to change all our Covalent components for `preservewhitespaces : false` because of this change?

Comment (christianmemije) [reply prettier/prettier.config.js:11]: Not quite. `preservewhitespaces` refers to preserving whitespace during build time.

Comment (richavyas) [reply prettier/prettier.config.js:11]: I understand that. was asking the question based on this statement: `In angular components, [preservewhitespaces] defaults to `false`, so just default to false as well.` if we are planning that for our components too.

Comment (christianmemije) [reply prettier/prettier.config.js:11]: `preserveWhitespaces` is used in many components, probably in some components that don't even require it. So we might be able to get rid of some usage. The only places that require it is when we use `<td-highlight/>`.  But even then, we might be better off using the `ngPreserveWhitespaces` directive or `<pre/>` tag instead.

**Review by jeremysmartt — APPROVED** [review]

**Pull request merged**